### PR TITLE
Added Cloudapp to the sales and marketing section

### PR DESCRIPTION
**Pull request by scotcha1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ put your business on the path to transformative growth. Disrupt your industry wi
 
 - [HubSpot](https://www.hubspot.com/startups) - Discover how HubSpot can help your startup grow with their CRM, sales, marketing, and customer service software, plus education and support.
 - [Drift](https://www.drift.com/startups/) - Drift is a leading conversational marketing platform and they're giving 86% off to early stage startups.
+- [CloudApp](https://www.getcloudapp.com/startups) - CloudApp offers a better way to acquire, engage and support your customers with quick shareable videos and images using an easy-to-use screen recorder. Receive 12 months of credit.
```
Comment (dakshshah96) [inline README.md]:
```suggestion
- [CloudApp Elevate](https://www.getcloudapp.com/elevate) - CloudApp offers a better way to acquire, engage and support your customers with quick shareable videos and images using an easy-to-use screen recorder. Get up to $1,000 CloudApp Credit for up to 12 months.
```

**Comment by scotcha1** [comment]
Thanks for updating the link! That looks great.

**Comment by dakshshah96** [comment]
@scotcha1 Can you please update the description to match the suggested changes? Thanks!

File: README.md (hunk 1 above)
Comment (dakshshah96) [inline README.md]: 👋  Hey, this should be good to merge if you can fix the following:

```suggestion
- [CloudApp Elevate](https://www.getcloudapp.com/elevate) - CloudApp offers a better way to acquire, engage and support your customers with quick shareable videos and images using an easy-to-use screen recorder. Get up to $1,000 CloudApp Credit for up to 12 months.
```

**Comment by dakshshah96** [comment]
Thanks for contributing 🎉

**Pull request merged**